portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>Yenessa's Portfolio</h1>
-      <p>This is my portfolio!! :D</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Ayyy</button>
-      <div id="greeting-container"></div>
-      <p>I'm currently a sophomore studying Computer Engineering at the University of Florida. I'm passionate about the societal impact of technology and the integration between hardware and software.</p>
-      <p>I also loooove cats! Here's one of my favorite cat images. </p>
-      <img src="images/catinbox.JPG"/>
+        <body style="background-color:thistle;">
+        <font face="garamond">
+            <h1>Yenessa's Portfolio</h1>
+            <a href="https://www.linkedin.com/in/yenavalos/" style="color:steelblue"><h4><i>Check out my LinkedIn page!</i></h4></a>
```

Comment:
Consider putting this inside a `<p>` paragraph tag. `<a>` is an inline element and may behave unpredictably if you add other content here.

Block elements will fill the full width of their container, inline elements will not.